cluster/common.sh

```diff
@@ -332,3 +332,24 @@ function find-release-tars() {
     exit 1
   fi
 }
+
+# Discover the git version of the current build package
+#
+# Assumed vars:
+#   KUBE_ROOT
+# Vars set:
+#   KUBE_GIT_VERSION
+function find-release-version() {
+    KUBE_GIT_VERSION=""
+    if [[ -f "${KUBE_ROOT}/version" ]]; then
+        KUBE_GIT_VERSION="$(cat ${KUBE_ROOT}/version)"
+    fi
+    if [[ -f "${KUBE_ROOT}/_output/full/kubernetes/version" ]]; then
+        KUBE_GIT_VERSION="$(cat ${KUBE_ROOT}/_output/full/kubernetes/version)"
+    fi
+
+    if [[ -z "${KUBE_GIT_VERSION}" ]]; then
+      echo "!!! Cannot find release version"
+      exit 1
```

Comment:
Yes, this function is meant to be fatal, just like finding the release. This will be used for tagging a release when we stage the pieces. If we can't find what to tag the release with, there's no point in continuing with staging the images. If we find a non-fatal usage for KUBE_GIT_VERSION, the next person along is welcome to refactor this method.